src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -22,6 +22,7 @@
     <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
     <Compile Include="System\Runtime\InteropServices\Attributes.cs" />
     <Compile Include="System\Runtime\InteropServices\ComAwareEventInfo.cs" />
+    <Compile Include="System\Runtime\InteropServices\ComEventInterfaceAttribute.cs" Condition="'$(TargetGroup)' != 'uapaot'" />
```

Comment:
I wasn't sure which side we wanted to go with this one so I just did the minimal to eliminate the conflict for building the mscorlib shim. ComAwareEventInfo didn't show up as a conflict for some reason.